src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -88,6 +101,26 @@ public boolean publish(String message, String color) {
         return result;
     }
 
+    private String getTokenToUse() {
+        if (credentialsAuthTokenId != null && !credentialsAuthTokenId.isEmpty()) {
+            StringCredentials credentials = lookupCredentials(credentialsAuthTokenId);
+            if (credentials != null) {
+                logger.info("Using secret for Slack auth token.");
+                return credentials.getSecret().getPlainText();
+            }
+        }
+
+        logger.info("Using plain text for Slack auth token.");
```

Comment:
The usage of the phrase "plain text" here is creating cognitive friction with the name of the function called on that other code path above, which is clearly named `getPlainText()`. Maybe the verbiage can get shifted to indicate "using Credentials secret" vs "not using Credentials secret"?